OrbitClientData/include/OrbitClientData/ProcessData.h

```diff
@@ -67,8 +68,8 @@ class ProcessData final {
   bool IsModuleLoaded(const std::string& module_path) const {
     return module_memory_map_.contains(module_path);
   }
-  static std::unique_ptr<ProcessData> Create(orbit_grpc_protos::ProcessInfo process_info);
-  std::unique_ptr<ProcessData> CreateCopy() const;
+
+  ProcessData CreateCopy() const;
```

Comment:
Deleting the copy constructor and copy assignment (that could just be default I think, as all the members are copyable) to then provide a dedicated method for the copy seems quite wrong.

Reply:
@antonrohr made this to ensure that there are no unintended copies. I am not sure, how relevant this with the recent changes and happy to remove this, if needed.

Reply:
Florian summarized it well, but I am happy to be convinced otherwise. 
ProcessData is passed around quite a bit and used as raw pointer in a lot of places. It is easy to accidentally make a copy when forgetting to use std::move.

Reply:
As the copy doesn't have any side effect, I don't see the problem of using the copy constructor and assignment. That's what they are for. The fact that it's used as raw pointer in many places doesn't have a connection with this. Using a dedicated method to prevent accidental copies seems a very weird pattern in C++. Based on this reasoning all containers (e.g., even `vector`s) should not have a copy constructor to prevent accidental copies.

Reply:
Okay. I guess the he explicit `ProcessData&&` that is now used in method signatures helps with enforcing move instead of copy. Lets remove the Create and use the default copy constructor/assingment